features/home/src/main/java/com/android254/droidconKE2020/home/ui/views/HomeFragment.kt

```diff
@@ -13,17 +13,17 @@ import coil.api.load
 import com.android254.droidconKE2020.core.di.browserModule
 import com.android254.droidconKE2020.core.utils.WebPages
 import com.android254.droidconKE2020.home.R
-import com.android254.droidconKE2020.home.databinding.FragmentHomeBinding
 import com.android254.droidconKE2020.home.di.homeRepositories
 import com.android254.droidconKE2020.home.di.homeViewModels
-import com.android254.droidconKE2020.home.domain.Speaker
 import com.android254.droidconKE2020.home.domain.Sponsor
 import com.android254.droidconKE2020.home.ui.adapters.*
 import com.android254.droidconKE2020.home.viewmodel.HomeViewModel
+import com.android254.droidconKE2020.speakers.models.Speaker
 import com.google.android.flexbox.FlexDirection
 import com.google.android.flexbox.FlexWrap
 import com.google.android.flexbox.FlexboxLayoutManager
 import com.google.android.flexbox.JustifyContent
+import kotlinx.android.synthetic.main.fragment_home.*
```

Comment:
Why did you revert to using kotin `synthetics`. Kindly lets be consistent and not mix up view binding in some areas and `synthetics` in others

Reply:
While trying to fix that offset upon launching home fragment, I tried a couple of things to point out the issue (including checking if some views were bound abnormally). I later realized it's related with rendering the sessions recyclerview and the entire view scrolling beneath the toolbar. I'll revert to data binding before making the final push

Reply:
Okay no problem